examples/binance-chain/index.tsx

```diff
@@ -1,26 +1,37 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import { render } from 'react-dom'
 import {
-  BinanceBscConnector,
+  BscConnector,
   UserRejectedRequestError,
 } from '@binance-chain/bsc-connector'
-import { ConnectionRejectedError, UseWalletProvider } from 'use-wallet'
+import {
+  ConnectionRejectedError,
+  useWallet,
+  UseWalletProvider,
+} from 'use-wallet'
 
 function App() {
+  const wallet = useWallet()
+  const [address, setAddress] = useState('')
+
+  useEffect(() => {
+    console.log(address)
+  }, [address])
+
   return (
     <div>
       <h1>Binance Chain Connector</h1>
-      <button>Connect</button>
+      <button onClick={() => { wallet.connect('injected'); setAddress(wallet.account)}}>Connect</button>
     </div>
   )
 }
 
 render(
   <UseWalletProvider
     connectors={{
-      bsc: {
+      injected: {
```

Comment:
```suggestion
      bsc: {
```

I think we should use `bsc` here, to not confuse it with the native `injected` connector.